.php_cs.dist

```diff
@@ -43,6 +43,7 @@ $config = PhpCsFixer\Config::create()
         'method_argument_space' => ['on_multiline' => 'ensure_fully_multiline'],
         'method_chaining_indentation' => true,
         'multiline_comment_opening_closing' => true,
+        'native_function_invocation' => false,
```

Comment:
to be dropped when number of open PR shrinks down a bit

we do follow Sf ruleset and add some more strict rules on top of it, but we do not exclude things from Sf ruleset.

Reply:
could be any good argument to let https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/3798 pass

Reply:
#3798 is indeed the one doing this removal (when it was adding it manually)

Reply:
indeed, silly me, forgot about it...
I added milestone there to not lose track of it